CMakeCommon.cmake

```diff
@@ -103,6 +103,10 @@ function (GenerateAddOnProject target acVersion devKitDir addOnName addOnSources
 
     find_package (Python COMPONENTS Interpreter)
 
+    if (APPLE)
+        set(CMAKE_INSTALL_RPATH "@loader_path/../Frameworks")
+    endif ()
```

Comment:
rpath beállítása mac-en. Bundle-nbelül keresi a dylib-eket / framework-öket.

A build során a gyártógépre mutató rpath-eket törli az install során.